src/main/webapp/app/shared/components/atoms/tag/tag.component.ts

```diff
@@ -15,6 +16,16 @@ export class TagComponent {
   icon = input<IconDefinition | undefined>(undefined);
   round = input<boolean>(false);
   iconRight = input<boolean>(false);
+  width = input<string | undefined>(undefined);
 
   readonly iconProp = computed(() => this.icon() as IconDefinition);
+
+  widthStyle = computed(() => {
+    if (this.width() != null) {
```

Comment:
nitpick: please check for undefined, not null, as width is either a string or undefined
```suggestion
    if (this.width() != undefined) {
```